plugins/css_loader.js

```diff
@@ -49,13 +50,19 @@ module.exports = new Plugin({
                 document.head.appendChild(window.customCss);
             }
             window.customCss.innerHTML = css;
+            if(window.glasscord)
+                ipc.send('glasscord_update');
             this.info('Custom CSS loaded!', window.customCss);
 
             if (window.cssWatcher == null) {
                 window.cssWatcher = fs.watch(cssPath, { encoding: 'utf-8' },
                 eventType => {
                     if (eventType == 'change') {
-                        readFile(cssPath).then(newCss => window.customCss.innerHTML = newCss);
+                        readFile(cssPath).then(newCss => {
+                            window.customCss.innerHTML = newCss;
+                            if(window.glasscord)
+                                ipc.send('glasscord_update');

```

Comment:
This one can be avoided by updating your observer--will DM

Reply:
Yeah, I am dumb